scripts/src/public/routes/orders.ts

```diff
@@ -115,3 +119,23 @@ export const getOrderHistory = async function(req: GetOrderHistoryRequest, res:
 	const orderList = await getOrderHistoryService(req.context.user!.id, filters, req.logger);
 	res.status(200).send(orderList);
 } as any as RequestHandler;
+
+export type WhitelistTransactionRequest = Request & {
+	params: {
+		order_id: string;
+	},
+	tx_envelope: string;
```

Comment:
where are these two coming from? how are they added to the request object?

Reply:
They get sent by the client as the json data in the post request. Though I didnt test with a client yet so im not sure it works

Reply:
well, then they don't exist on the request object, they need to be in the `req.body`